lineapy/plugins/jinja_templates/dvc_dag_PythonOperator.jinja

```diff
@@ -0,0 +1,18 @@
+import {{ MODULE_NAME }}
+import pickle
+
+def task_{{ STAGE["name"] }}():
```

Comment:
Question here: 

Can we utilize the task rendering template that we already have or is there is custom logic in this block? 

https://github.com/LineaLabs/lineapy/blob/main/lineapy/plugins/jinja_templates/task/task_function.jinja 

E.G this is how airflow uses it. 

https://github.com/LineaLabs/lineapy/blob/d3a0f533ee7a44a4002fbfb86faddd54429b84b5/lineapy/plugins/airflow_pipeline_writer.py#L223 

And an example of a rendered task: 
https://github.com/LineaLabs/lineapy/blob/d3a0f533ee7a44a4002fbfb86faddd54429b84b5/tests/unit/plugins/__snapshots__/test_writer.ambr#L983


Right now this template is using 
`{{ MODULE_NAME }}.get_{{ out }}` instead of whatever the call block, which actually gets the wrong function sometimes. 

For example in the housing example, this template generates 

`training_data = data_housing_pipeline_module.get_training_data()`
instead of 
`training_data = data_housing_pipeline_module.get_cleaned_data_housing()`

Reply:
> I wasn't sure how the rendered method name is generated.

TaskDefinition has a call block which calls the right function in the module so you don't have to know the exact function name your task needs to call.

I think the only piece where this template needs to differ from the current task template is the if main piece which can be added to the current [task template](https://github.com/LineaLabs/lineapy/blob/d3a0f533ee7a44a4002fbfb86faddd54429b84b5/lineapy/plugins/jinja_templates/task/task_function.jinja). 

```
if __name__ == "__main__":
    task_{{ STAGE["name"] }}()
```

I'd like to reuse this template as much as possible.

Reply:
> fix the path for the pickled objects

This one is a bit trickier, the current [task template](https://github.com/LineaLabs/lineapy/blob/d3a0f533ee7a44a4002fbfb86faddd54429b84b5/lineapy/plugins/jinja_templates/task/task_function.jinja) has places for you to insert loading and dumping blocks. 

Currently for airflow these are generated by the [render_task_io_serialize_blocks function](https://github.com/LineaLabs/lineapy/blob/b77dba5737ebbfbbf6bd2cfc154717b5a39824e3/lineapy/plugins/task.py#L120) 

which uses the following template to control where to serialize to. https://github.com/LineaLabs/lineapy/tree/d3a0f533ee7a44a4002fbfb86faddd54429b84b5/lineapy/plugins/jinja_templates/task/localpickle 

If you want to serialize to a different place not under `tmp`, I'd prefer to use a similar workflow of defining new templates for the loading and dumping blocks and passing those into the current task rendering template.

Reply:
As discussed in our conversation on slack, we will be skipping the merging of these two templates for this PR.
Tracking interally as https://linea.atlassian.net/browse/LIN-736